x-pack/legacy/plugins/uptime/public/hooks/use_url_params.ts

```diff
@@ -20,7 +20,10 @@ export const useUrlParams = (
   const currentParams: any = qs.parse(search[0] === '?' ? search.slice(1) : search);
 
   const updateUrl = (updatedParams: any) => {
-    const updatedSearch = qs.stringify({ ...currentParams, ...updatedParams });
+    const updatedSearch = qs.stringify({
```

Comment:
Interesting this formatting changed across release 🤷‍♂ . Not a problem, just curious.